concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/BakerSetFinalizationRewardCommission.java

```diff
@@ -0,0 +1,27 @@
+package com.concordium.sdk.responses.transactionstatus;
+
+import com.concordium.sdk.transactions.AccountAddress;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class BakerSetFinalizationRewardCommission extends AbstractBakerResult {
+    // parts per 100000
+    private final int finalizationRewardCommission;
+
```

Comment:
It would be good to have a type for PartsPerHundredThousand.

THis is very fragile.